nixos/release.nix

```diff
@@ -222,6 +222,20 @@ in rec {
     }).config.system.build.amazonImage)
 
   );
+  amazonImageZfs = forMatchingSystems [ "x86_64-linux" "aarch64-linux" ] (system:
```

Comment:
This is a bit misleading, I have not even tried this on aarch64.

Reply:
Since this is ready-for-review, I think now is a good time to try this... ;)

Reply:
Confirmed.